source/manual/alerts/rebooting-machines.html.md

```diff
@@ -103,6 +103,34 @@ more info.
 
 ## Special cases
 
+### Rebooting Jenkins CI agents
+
+Sometimes a CI agent starts continually erroring on Jenkins jobs, and the
+most straightforward way of fixing the issue is to reboot the machine.
+
+First, visit the [Jenkins nodes list](https://ci.integration.publishing.service.gov.uk/computer/).
+Click on the problematic agent and then "Mark this node temporarily
+offline". You'll need to provide a reason, which could just be "to reboot
+problematic agent". This will stop the agent from being used for new jobs.
+
+SSH into the agent; the machine number to SSH into will match the agent
+number. For example:
+
+```sh
+gds govuk connect ssh -e integration ci_agent:6
+```
+
+Reboot the machine: `sudo reboot`.
+
+It will take a few minutes for Jenkins to start up on the agent. You can
+SSH back into the machine and monitor this if you wish: use `top` to list
+the running processes, then `>` to sort by memory usage. You should see a
+`java` process at the top of the list when Jenkins is running.
+
```

Comment:
Since we noticed that we seem to have Jenkins configured to ssh into the node and start the agent rather than the agents registering with the master, I'm not sure whether this step is meaningful.

Reply:
I've now removed that paragraph 👍 thanks Chris.